.github/workflows/build_apk.yml

```diff
@@ -0,0 +1,38 @@
+name: Build APK
+
+on:
+  push
+
+jobs:
+  build-apk:
+
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22.x'  
+          cache: 'npm'
+
+      - run: npm ci
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          distribution: 'temurin'  
+          java-version: '11'
+
+      - name: Install Android SDK
+        run: sudo apt-get install -y android-sdk
+
```

Comment:
You miss a command to create an Android folder from the project.

Reply:
Yes I am trying to fix it! Thanks Margaux